### PR TITLE
Designs for Inquiry emails

### PR DESCRIPTION
**Pull request by BenjaminSsempala**
**_WHAT DOES THIS PR DO?_**
- Designs for Inquiry emails

**_WHAT ISSUES ARE RELATED TO THIS PR?_**

- Jira cards
    - []()
- GitHub issues
    - Closes #<enter GitHub issue number here>

**_HOW DO I TEST OUT THIS PR?_**

**_WHICH ENDPOINTS SHOULD BE READY FOR TESTING?:_** 

**_IS THERE ANY JENKINS CONSOLE LINK FOR CODE COVERAGE AND BUILD INFO?_**

**_ARE THERE ANY RELATED PRs?_**

### CODE REVIEW

**Review by Baalmart — COMMENTED** [review]
thanks @BenjaminSsempala , please remember to also add unit tests for this.

As you go about this, do not be worried about the tests passing, just add them and make sure they at least run even if they fail. :)

Great work!

**Comment by BenjaminSsempala** [comment]
> thanks @BenjaminSsempala , please remember to also add unit tests for this.
> 
> As you go about this, do not be worried about the tests passing, just add them and make sure they at least run even if they fail. :)
> 
> Great work!

@Baalmart I've also added unit tests for this

File: src/auth-service/config/emailConstants.js [hunk 1]
```diff
@@ -0,0 +1,125 @@
+const emailConstants = {
+    greetings: (name) => {
+        return `<tr>
+                                <td
+                                    style="padding-bottom: 24px; color: #344054; font-size: 16px; font-family: Inter; font-weight: 600; line-height: 24px; word-wrap: break-word;">
```
Comment (Baalmart) [inline src/auth-service/config/emailConstants.js:5]: Hi @BenjaminSsempala , in a bid to minimise the number of new files, can't these be added to the default config object of the `constants` file?

**Review by Baalmart — APPROVED** [review]
thanks @BenjaminSsempala , just left one small comment

**Review by Baalmart — APPROVED** [review]
thanks @BenjaminSsempala

**Pull request merged**